gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -49,13 +54,21 @@ interface GalleryExampleTileProps extends GalleryBaseProps {
 // Component for a simple nx-tile with a specified title and contents
 export const GalleryTile: FunctionComponent<GalleryTileProps> =
   function GalleryTile({ className, id, title, actionButtons, children }) {
-    const tileClasses = classnames('nx-tile', className);
+    const tileClasses = classnames('nx-tile', className),
+        routeParams = useParams<{ pageName: string }>();
 
     return (
       <div id={id} className={tileClasses}>
         <div className="nx-tile-header">
           <div className="nx-tile-header__title">
             <h2 className="nx-h2">{title}</h2>
+            { id && (
```

Comment:
are the smooth brackets surrounding NxTooltip necessary?

Reply:
> are the smooth brackets surrounding NxTooltip necessary?

No but they match the parentheses that we typically put around multi-line return statements (like the one starting on line 61 that this line is part of)